src/test/util/setup_common.h

```diff
@@ -80,6 +81,7 @@ struct BasicTestingSetup {
     ~BasicTestingSetup();
 
     const fs::path m_path_root;
+    ArgsManager m_args_manager;
```

Comment:
In commit "BasicTestingSetup: Add ArgsManager." (d4f86a3f608c98365b537d78d7ef1693463d7b9b)

Convention other places is to call `ArgsManager` instances `args`, suggest naming this `m_args`

Reply:
Applied, thanks.